salsah/src/public/vendor/ckeditor-4.7.1-Editor/config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
:) I don't remember that conversation either!

the use of this config file is discussed here, incidently on a page about `format_tags`: https://docs.ckeditor.com/#!/guide/dev_format

This is as much code as config, it defines the method:
```js
CKEDITOR.editorConfig = function( config ) {
```

Making it used by all the CKEditor instances.

We can overwrite this method in our code too with the same effect with less risks of forgetting to keep this file when upgrading ckeditor.

My experience with add-ons that use *jQuery* is that some code needs to run in a general config (`CKEDITOR.editorConfig = function( config ) {`).  
I did not investigate whether it is different run time or javaScript variable scope that requires that.

The config is for this instance only when we define a `config` variable passed to 
```js
CKEDITOR.replace(textarea[0], config);
```